doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/01_Within_V5/README.md

```diff
@@ -1,6 +1,9 @@
 # Upgrade Notes for Upgrades within Pimcore 5
 
 ## Version 5.8.0
+- Add PathFormatter interface, static path formatters are now deprecated and will be removed with Version 6.0. You can read
+ more about that [here](../../../05_Objects/01_Object_Classes/05_Class_Settings/14_Path_Formatter.md).
+- Removed method \Pimcore\Model\DataObject\ClassDefinition\Data::setFieldtype($fieldtype)
```

Comment:
please remove duplicate docs :)

Reply:
I don't see any duplicate here?

Reply:
It's about "setFieldtype" notice already mentioned in 5.6.0

Reply:
done